guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -83,7 +80,9 @@ To use them inside an EXISTING $FOREMAN_PROXY, run this command INSTEAD:
 ----
 endif::[]
 
-. On {ProjectServer}, from the output of the `katello-certs-check` command, depending on your requirements, enter the `{certs-generate}` command that generates a certificate for a new or existing {SmartProxy}.
+. Depending on your requirements, enter the `{certs-generate}` command on the {ProjectServer} that generates a certificate for a new or existing {SmartProxy}.
+The output of the `katello-certs-check` command may not be accurate in some cases.
+Therefore, you must follow the steps mentioned above instead of command outputs.
```

Comment:
It'd be great to have a bug reference. For example:
```suggestion
// https://bugzilla.redhat.com/show_bug.cgi?id=XYZ
Therefore, you must follow the steps mentioned above instead of command outputs.
```

That way you easily find out in the future if it's still a problem.

Reply:
Thanks for this suggestion. Wasn't really aware if we could add it for internal reference.